assets/src/components/detours/detourMap.tsx

```diff
@@ -32,7 +33,7 @@ interface DetourMapProps {
   /*
    * Stops along the original route shape
    */
-  stops: Stop[]
+  stops: (Stop & { missed: boolean })[]
```

Comment:
I'm on the fence about this. On the one hand, I _really_ hate inlined type definitions, since they get complicated and tough to read. 

On the other hand, this isn't actually that complicated, and creating a bespoke type for this (like `DetourMapStop` or something like that), feels a bit like overkill.

Reply:
I was also on the fence about this, but it works and it's not that difficult to read, so I think it's fine.